### DISCUSSION TITLE
Webhook for gotify

### DISCUSSION THREAD

**Discussion by GerritKuilder** [henrygd/beszel#189] [Support] [answered]
Hello All,

just started using this and the only snag I now run into is that I want to send alerts to my gotify server, I can see that I need to create a webhook but I am not sure if I do need to do something on the gotify side too or how a webhook should look, the generec://mypus.domain.com  plus other swicthes added all result in failurer (4x errors)
Can anybody guide me to to a working example?

Kind regards,

Gerrit

**Comment by henrygd** [comment, marked as answer]
Hi Gerrit,

Please see this link for instructions on formatting the URL for Gotify:

https://containrrr.dev/shoutrrr/v0.8/services/gotify/

Note that it has a dedicated URL schema `gotify://`, so do not use `generic://`.

**Reply by GerritKuilder** [reply]
Thank you that did the trick.

**Reply by hs6666** [reply]
hi, just saw this

can you send me the url format of your setup please so i can see what i am doing wrong connecting to my gotify server?

thanks

**Reply by GerritKuilder** [reply]
A) create a new app for the token, not a lient token
b) gotify://<gotifyUrl>/gotifyToken>
